manifest.json

```diff
@@ -10,12 +10,37 @@
         "targetDirPrefix": "backports.functools_lru_cache-"
       },
       "bleach": {
-        "version": "1.2.2",
-        "downloadFormat": "zip",
-        "url": "https://github.com/jsocol/bleach/archive/v1.2.2.zip",
+        "version": "3.1.0",
+        "downloadFormat": "tar",
+        "url": "https://github.com/mozilla/bleach/archive/v3.1.0.tar.gz",
```

Comment:
Couldn't we use the zip version of the library, like we did before? Ditto below.

Reply:
Yeah, sure. I will switch to zip.